packages/density/density.py

```diff
@@ -0,0 +1,94 @@
+import json
+import requests
+import os, sys
+import heapq
+
+auth_key = os.environ['DENSITY_API_KEY']
+types_of_places= {'eat':["JJ's Place", "John Jay Dining Hall", "John Jay"], 'study': ["Avery", "Butler", "Lehman Library", "Lerner", "Northwest Corner Building", "East Asian Library", "Uris"]}
+
+def density_msg(result):
+	print(result)
+
+	try:
+		location = result['parameters']['density_entities']
+	except:
+		location = 'NoCo'
+
+	return parse_json(location) 
+
+def parse_json(location):
+	url = 'http://density.adicu.com/latest?auth_token='+auth_key
```

Comment:
Maybe you should make the actual API request in a separate method - the method says its parsing a json so it might be better to leave additional logic like that in another method